src/store/reducers/healthcheckInfo.ts

```diff
@@ -110,6 +109,26 @@ const getInvertedConsequencesTree = ({
         : [];
 };
 
+const getIssuesMap = (data: IssueLog[]): Record<StatusFlag, number> => {
+    const issuesMap = {} as Record<StatusFlag, number>;
+
+    for (const issue of data) {
+        if (!issuesMap[issue.status]) {
+            issuesMap[issue.status] = 0;
+        }
+        issuesMap[issue.status]++;
+    }
+
+    const sortedStatusFlags: StatusFlag[] = _flow([
+        _sortBy((status: StatusFlag) => mapStatusToPriority[status]),
+    ])(Object.keys(issuesMap));
+
+    return sortedStatusFlags.reduce((obj, key) => {
+        obj[key] = issuesMap[key];
+        return obj;
+    }, {} as Record<StatusFlag, number>);
+};
+
```

Comment:
Do you really need to prepare statistics as `Record<StatusFlag, number>`? It seems you don't use it anywhere else and transform before render. So you can just return `[StatusFlag, number][]` and use it as it is in component
```
return Object.entries(issuesMap).sort(sortFunc)
```